src/main/java/Duke/Event.java

```diff
@@ -0,0 +1,27 @@
+package Duke;
+
+class Event extends Task{
+    String description;
+    String at;
+    Event(String description, String at) {
+        this.description = description;
+        this.at = at;
+    }
+    @Override
+    void initializeMessage() {
+        System.out.println("Got it! I've added this task:");
+        displayMessage();
+        System.out.println("Now you have "+ TaskManager.noOfTasks +" tasks in the list.");

```

Comment:
Operators should be surrounded by a space character, as per the Java coding standard.